ledgersmb/start.sh

```diff
@@ -27,6 +27,8 @@ fi
 PERL5LIB=lib
 export PERL5LIB
 
+make dojo
```

Comment:
Please make this conditional on the build not having been executed before: I regularly need to restart the container after i change Perl code; when i change JS code, the container doesn't need restarting... So this runs `make dojo` exactly when I'm likely not to need it.

Reply:
Done